src/file/upload.repository.ts

```diff
@@ -1,5 +1,6 @@
 import { IUpload } from './upload.interface';
 import { uploadModel } from './upload.model';
+import { ObjectID } from 'mongodb';
```

Comment:
change to mongoose

Reply:
mongoose doesn't seem to export this type. it uses mongodb's type in its core: 
![image](https://user-images.githubusercontent.com/23077804/59275401-ebab4880-8c64-11e9-88df-442ad23bc0f4.png)

Reply:
you used it already.
mongoose.Types.ObjectId()  ?

Reply:
this syntax doesn't work there

Reply:
This is the correct way to import and use `ObjectID`.